src/main/resources/db/migration/V1.9__create_feedback_request_entry_table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS feedback_request_entry (
+    id BIGINT GENERATED BY DEFAULT AS IDENTITY NOT NULL,
+    feedback_request_id BIGINT NOT NULL,
+    question_id BIGINT NOT NULL,
```

Comment:
there should be a composite unique key for `feedback_request_id` & `question_id`